pkg/apis/v1/nodepool_status.go

```diff
@@ -27,6 +27,9 @@ const (
 	ConditionTypeValidationSucceeded = "ValidationSucceeded"
 	// ConditionTypeNodeClassReady = "NodeClassReady" condition indicates that underlying nodeClass was resolved and is reporting as Ready
 	ConditionTypeNodeClassReady = "NodeClassReady"
+	// ConditionTypeHealthy = "Healthy" condition indicates when the nodepool has more 80% of nodes in an healthy state
+	// This status condition is Alpha status condition
+	ConditionTypeHealthy = "Healthy"
```

Comment:
I think we still need to mark that this is an "alpha" status condition that is associated with the NodeRepair feature

Reply:
Discussed offline, dropping the observability aspect from this PR and revising at a later time